packages/graphql/src/configs/localhost.js

```diff
@@ -41,8 +41,8 @@ const config = {
   tokens: [],
   messagingAccount: '0xBfDd843382B36FFbAcd00b190de6Cb85ff840118',
   messaging: {
-    messagingNamespace: 'origin',
-    globalKeyServer: 'http://localhost:6647'
+    messagingNamespace: 'origin:dev',
+    globalKeyServer: 'https://messaging.dev.originprotocol.com'
```

Comment:
Just curious, we do not want the localhost to point to the local running messenger container?

Reply:
For docker, it would still point to localhost:6647. This config is used only when the services are **not** started with docker.

Reply:
Yeah that makes sense 👍